lib/python/rose/config_editor/data_helper.py

```diff
@@ -434,16 +440,23 @@ def get_ns_ignored_status(self, namespace):
         status_counts = object_statuses.items()
         status_counts.sort(lambda x, y: cmp(x[1], y[1]))
         if not status_counts:
```

Comment:
It seems as though `cache[namespace]` is being set on a few more lines than it really needs to be here (actually above this point as well but it's clearest in this section) - for instance it could appear once after L445 instead of on L447,L450 and L452